sympy/functions/special/gamma_functions.py

```diff
@@ -1002,6 +1002,21 @@ class digamma(Function):
 
     In this case, ``digamma(z) = polygamma(0, z)``.
 
+    >>> from sympy import digamma
+    >>> digamma(0)
+    zoo
+    >>> from sympy import Symbol
+    >>> z = Symbol('z')
+    >>> digamma(z)
+    polygamma(0, z)
+
+    To retain digamma as it is:
+
+    >>> digamma(0, evaluate=False)
+    digamma(0)
+    >>> digamma(z, evaluate=False)
+    digamma(z)
+
```

Comment:
The examples should better be put under the section `Examples` in numpydoc format. Same for below.
```
Examples
========
```